idd/Energy+.idd.in

```diff
@@ -6657,22 +6677,22 @@ WindowMaterial:Glazing:EquivalentLayer,
        \maximum 1.0
        \autocalculatable
        \default autocalculate
-  N23, \field Infrared Transmittance (applies to front and back)
+  N23, \field Thermal Transmittance (applies to front and back)
        \note The long-wave hemispherical transmittance of the glazing.
        \note Assumed to be the same for both sides of the glazing.
        \type real
        \units dimensionless
        \minimum 0.0
        \maximum 1.0
        \default 0.0
-  N24, \field Front Side Infrared Emissivity
+  N24, \field Front Side Thermal Emissivity
        \note The front side long-wave hemispherical emissivity of the glazing.
        \type real
        \units dimensionless
        \minimum> 0.0
        \maximum< 1.0
        \default 0.84
-  N25; \field Back Side Infrared Emissivity
+  N25; \field Back Side Thermal Emissivity
```

Comment:
No.  I hate to suggest undoing most of the work for [this particular item](https://github.com/NREL/EnergyPlus/issues/5896#issuecomment-259998883) but it's probably best to keep the field names as they were (infrared) and just change the docs to match.